### ISSUE TITLE
WhitespaceAfter should support `ELLIPSIS` token

### ISSUE THREAD

**Issue by nmancus1** [checkstyle/checkstyle#11116]
I have downloaded the latest cli from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

Check documentation at https://checkstyle.sourceforge.io/config_whitespace.html#WhitespaceAfter

**How it works Now:**

```bash

➜  javac Test.java
➜  cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
        <module name="WhitespaceAfter">
          <property name="tokens" value="ELLIPSIS"/>
        </module>
    </module>
</module>

➜  cat Test.java  
public class Test {
    void method1(int...myVarargs) { // expected violation
    
    }
    void method2(int... myVarargs) { // ok
    
    }
}

➜  java -jar checkstyle-9.2-all.jar -c config.xml Test.java
com.puppycrawl.tools.checkstyle.api.CheckstyleException: cannot initialize module TreeWalker - Token "ELLIPSIS" was not found in Acceptable tokens list in check com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck
        at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:473)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:201)
        at com.puppycrawl.tools.checkstyle.Main.runCheckstyle(Main.java:405)
        at com.puppycrawl.tools.checkstyle.Main.runCli(Main.java:332)
        at com.puppycrawl.tools.checkstyle.Main.execute(Main.java:191)
        at com.puppycrawl.tools.checkstyle.Main.main(Main.java:126)
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Token "ELLIPSIS" was not found in Acceptable tokens list in check com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck
        at com.puppycrawl.tools.checkstyle.TreeWalker.registerCheck(TreeWalker.java:223)
        at com.puppycrawl.tools.checkstyle.TreeWalker.setupChild(TreeWalker.java:133)
        at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:201)
        at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:468)
        ... 5 more
Checkstyle ends with 1 errors.

```

However, `WhitespaceAroundCheck` does support `ELLIPSIS`. But, this is not a frequently used style. In fact, the [JLS](https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html) specifically mentions that

> Note that the ellipsis (...) is a token unto itself (§3.11). It is possible to put whitespace between it and the type, but this is discouraged as a matter of style.

**Is your feature request related to a problem? Please describe.**
I would expect `...` to be a valid token for `WhitespaceAfterCheck`, and for it to enforce style noted above.

**Describe the solution you'd like**

```bash
[ERROR] Test.java:2:21: '...' is not followed by whitespace. [WhitespaceAfter]
```

**Additional context**

Noticed at https://github.com/checkstyle/checkstyle/pull/11094#discussion_r775249437, WhitespaceAfterCheck does not currently support `ELLIPSIS` token.

`ELLIPSIS` should be a default token for `WhitespaceAfterCheck`. Since we use default module for this check in `checkstyle_checks.xml`, we may find some violations in the project.

**Comment by pbludov** [comment]
Some teams may prefer whitespace around style:

https://github.com/treeform/orekit/blob/master/src/main/java/org/orekit/data/PolynomialParser.java#L256

**Comment by strkkk** [comment]
> void method1(int...myVarargs) { // expected violation

in case this code is compliable (which is surprice for me), https://checkstyle.org/config_whitespace.html#NoWhitespaceAfter check can be enhanced with ellipsis token as well (not default ofc)

**Comment by nmancus1** [comment]
> Some teams may prefer whitespace around style...

@pbludov do you think we should not make `ELLIPSIS` a default token?

**Comment by pbludov** [comment]
> @pbludov do you think we should not make `ELLIPSIS` a default token?

Let's make it and look at the regression tests.

**Comment by Vyom-Yadav** [comment]
On it.
